Tests/Command/DoctrineCommandTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Doctrine\Bundle\MigrationsBundle\Tests\DependencyInjection;
+
+use Doctrine\Bundle\MigrationsBundle\Command\DoctrineCommand;
+use Doctrine\DBAL\Migrations\Configuration\Configuration;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBag;
+
+class DoctrineCommandTest extends \PHPUnit\Framework\TestCase
```

Comment:
`\PHPUnit_Framework_TestCase` might be better

Reply:
Maybe it's better to leave it as it is and require PHPUnit 4.8.35+.

Reply:
Maybe it's even better to import the class namespace. :wink: